src/networks/abc/checkpoints.cpp

```diff
@@ -7,9 +7,13 @@
 
 static CCheckpointData mainNetCheckpointData = {
     .mapCheckpoints = {
+        // Exodus activation
+        {143509, BlockHash::fromHex("00000000000df2347fd05d10e9ce5846f3089ef7e6"
```

Comment:
Checkpoint the block with the upgraded rules, which is one later

Reply:
Technically not true. Only the funding addresses update one later. Any other rules will activate at this block.

Reply:
This is the code for the funding addresses:

```

static const std::vector<std::string> &
GetPayoutAddresses(const Consensus::Params &params,
                   const CBlockIndex *pindexPrev) {
    if (IsLeviticusEnabled(params, pindexPrev)) {
        return params.coinbasePayoutAddresses.leviticus;
    }
    if (IsExodusEnabled(params, pindexPrev)) {
        return params.coinbasePayoutAddresses.exodus;
    }
    return params.coinbasePayoutAddresses.genesis;
}
```

`IsExodusEnabled` and `IsLeviticusEnabled` will be used exactly the same for all other rule updates, no?